md/bonita-bpm-overview.md

```diff
@@ -1,12 +1,12 @@
 # Bonita platform overview
```

Comment:
Now we need to remove this page from the repository. How can we make sure no links navigate to this page?